packages/connectivity/src/scp-cf/destination/destination-accessor-provider-subscriber-lookup.spec.ts

```diff
@@ -158,17 +159,76 @@ function assertMockUsed(mock: nock.Scope, used: boolean) {
 }
 
 describe('JWT type and selection strategies', () => {
-  beforeEach(() => {
+  beforeEach(async () => {
     mockServiceBindings();
     mockVerifyJwt();
     mockServiceToken();
+    await destinationCache.clear();
   });
 
   afterEach(() => {
     nock.cleanAll();
     jest.clearAllMocks();
   });
 
+  it('should use cache by default', async () => {
```

Comment:
[req] The destination-cache.spec.ts already checks all behavior regarding `useCache: true`. We should update those tests (not pass this property anymore) and test that they are still green. There are also tests for the other behavior where nothing should be cached. 

Please remove these tests here.